workers/unity/Assets/Playground/Scripts/UI/ConnectionScreenController.cs

```diff
@@ -32,6 +32,7 @@ private void TryConnect()
 
             if (workerConnector == null)
             {
+                errorMessage.text = "Failed to create a mobile worker connector.";
```

Comment:
Can improve this message: the actual problem is that the worker connector behaviour could not be found on the client worker prefab

Reply:
I am not sure if we should dive into that much detail in the error message, considering it's shown in the UI.

Reply:
Anything that will help the developer resolve the problem. From this message by itself, what can they do to fix it? An alternative is to also log an error so they can use the stack trace from the editor.

Reply:
Should it be throwing an exception, rather than simply logging?

Reply:
Exception sounds better.

Also destroy the worker before throwing the exception :)